docker-compose.override.dist

```diff
@@ -0,0 +1,8 @@
+# Copy this file to docker-compose.override.yml
```

Comment:
Could we move the `docker-compose` work out into another PR please? It needs some tests around it too please : ]